### DISCUSSION TITLE
[Bug] Invalid Prompt data

### DISCUSSION THREAD

**Discussion by mikejernil** [webstudio-is/webstudio-community#70] [Q&A]
Invalid Prompt data whatever I give

ex- create a header with profile picture
create a header with logo
create a testimonial section in header

**Comment by kof** [comment]
That's so weird, literally just took your prompt and tried: https://share.descript.com/view/LKRlxm7sY61

**Reply by kof** [reply]
Are you trying it in our cloud?

**Reply by kof** [reply]
Can you share a video? and a project share link?
